app/src/main/java/org/simple/clinic/ageanddateofbirth/DateFormatterModule.kt

```diff
@@ -0,0 +1,22 @@
+package org.simple.clinic.ageanddateofbirth
+
+import dagger.Module
+import dagger.Provides
+import org.simple.clinic.di.AppScope
+import org.threeten.bp.format.DateTimeFormatter
+import java.util.Locale
+import javax.inject.Named
+
+@Module
+class DateFormatterModule {
```

Comment:
Injecting these formatters is a good idea 👍

Reply:
🙏